File: backend/src/main/java/com/liveticket/event/entity/Event.java
package com.liveticket.event.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@TableName("lt_event")
public class Event {

    private Long id;
    private String title;
    private String artist;
    private String category;
    private String description;
    private String cityCode;
    private String venueName;
    private String address;
    private BigDecimal longitude;
    private BigDecimal latitude;
    private String coverUrl;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private LocalDateTime saleStartTime;
    private LocalDateTime saleEndTime;
    private Integer status;
    private Long heatScore;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: frontend/src/types/event.ts
export interface EventVO {
  id: number
  title: string
  artist: string
  category: string
  cityCode: string
  venueName: string
  address: string
  coverUrl: string
  startTime: string
  endTime: string
  saleStartTime: string
  saleEndTime: string
  status: number
  heatScore: number
  minPrice?: number | null
}

export interface EventDetailVO extends EventVO {
  description: string
}

export interface TicketSkuVO {
  id: number
  eventId: number
  skuName: string
  price: number
  totalStock: number
  availableStock: number
  perUserLimit: number
  status: number
}

export interface NearbyEventVO extends EventVO {
  distanceKm: number
}
